rollup.config.js

```diff
@@ -0,0 +1,42 @@
+import typescript from 'rollup-plugin-typescript2';
+import { terser } from "rollup-plugin-terser";
+import * as path from "path";
+import * as fs from "fs";
+
+const BASE_PATH = process.cwd();
+const getConfigFile = (name) => {
+  const filePath = path.resolve(BASE_PATH, name);
+  if (fs.existsSync(path.resolve(BASE_PATH, name))) {
+    return filePath;
+  }
+
+  return path.resolve(BASE_PATH, 'node_modules', '@stoplight', 'scripts', name);
+};
+
+let projectRollupConfig = null;
+
+try {
+  ({ default: projectRollupConfig } = require("esm")(module)(path.resolve(process.cwd(), 'rollup.config.js')));
+} catch (ex) {
+}
```

Comment:
Let's at least log an error message see we know what's going on.

Reply:
Why would we? That config is optional.

Reply:
BTW, I'm not entirely sure whether the concept of extending config is properly implemented here.
I believe we should always prefer local config over the shared one.
If one wants to extend the shared config, it can be imported and merged. We do something similar for tsconfig.
Thoughts?

Reply:
Yeah... would it be better to add a `tsconfig.json` to the `templates` and have _that_ import the root `tsconfig`? (similarly to how we do tslint config?)

And the same with rollup, have a local rollup.config.json in `templates` and have _that_ import the shared rollup.config?

Reply:
Then you wouldn't need `esm` at all.

Reply:
![image](https://user-images.githubusercontent.com/9273484/57285498-a028dc00-70b3-11e9-8742-f64bada11bd5.png)
eh, we still do need esm :(

Reply:
Luckily, rollup handles cjs just fine, so rewrote the shared config.